examples/aws-noaa-oisst-avhrr-only-example.ipynb

```diff
@@ -0,0 +1,936 @@
+{
```

Comment:
<p>Needs a top level title and intro that is about the Dataset before explaining that this dataset is accessible thanks to kerchunk and pangeo.</p><p> </p><p>In some of the later sections the Cloud Optimized information comes first, but should probably come after the description of what the user is actually trying to do. Overall the orientation should be more focused on SST data with the technical parts there to explain how Cloud Optimized access works. While this is the first example of the Kerchunk method we expect to do many.</p><p> </p><p>Also added suggested Comment lines for some of the code blocks, to make sure people know what each one does.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/asdi-examples/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>